src/Views/Form/Templates/Classic/resources/js/form.js

```diff
@@ -101,11 +104,75 @@ function splitDonationLevelAmountsIntoParts() {
                 h('span', {className: "give-amount-decimal"}, decimalForAmount),
             ),
             // There’s an intentional leading space before the currency symbol.
-            currency.symbolPosition === 'after' && ` ${h(CurrencySymbol, {position: 'after'})}`,
+            //currency.symbolPosition === 'after' && ` ${h(CurrencySymbol, {position: 'after'})}`,
```

Comment:
@knowler I forgot that I commented this out. There were some issues with displaying the currency. Can you uncomment this and see what is going on?

Reply:
Resolved in a68dbd9

I think the `h` function just didn’t like the padded string. I realized that all that stuff is just visual and not going to be read by a screen reader so the space wasn’t important to keep.